src/Makefile.am

```diff
@@ -111,7 +111,7 @@ souffle_sources = \
         RamIndexAnalysis.cpp  RamIndexAnalysis.h  \
         InlineRelationsTransformer.cpp            \
         LogStatement.h                            \
-        InterpreterIndex.h            InterpreterIndex.cpp	\
+        InterpreterIndex.h            InterpreterIndex.cpp \
```

Comment:
Would be nice to have a formatting check so we didn't keep introducing these changes. Another PR, though.